bin/qpid_install.sh

```diff
@@ -23,20 +11,13 @@ cd $HOME/build
 git clone --branch 0.18.0-rc1 https://github.com/apache/qpid-proton.git
 cd qpid-proton
 
-# There is a strange dependency on JSON that we need to change to version of
-# at least 2; otherwise there will be a conflict.
-sed -i .bak -e 's/ 0/ 2/' proton-c/bindings/ruby/qpid_proton.gemspec.in
-
 # Configure the source of Qpid Proton.
 mkdir build
 cd build
-cmake .. -DCMAKE_INSTALL_PREFIX=$HOME/pkg -DSYSINSTALL_BINDINGS=OFF
+cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_BINDINGS= -DSYSINSTALL_BINDINGS=OFF
```

Comment:
Once you remove bindings (yay, btw!), you can also remove swig

Reply:
Yes, I will create a separat PR for that. Thanks!